.jshintrc

```diff
@@ -1,4 +1,5 @@
 {
+  "esversion": 6,

```

Comment:
I don't see any other option. Using [`module`](http://jshint.com/docs/options/#module) fails with `The 'module' option is only available when linting ECMAScript 6 code.`